src/main/java/uk/gov/hmcts/reform/hmc/controllers/PartiesNotifiedController.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.hmc.controllers;
+
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.hmc.service.HearingManagementService;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@Validated
+public class PartiesNotifiedController {
+
+    protected static final Logger logger = LoggerFactory.getLogger(PartiesNotifiedController.class);
```

Comment:
```suggestion
```
Not being used? (Can also use Lombok's `@Slf4j` when we do need to as well to remove the need for this declaration)

Reply:
Ooh. Thanks!